iquip/apps/builder.py

```diff
@@ -291,13 +291,16 @@ def value(self) -> Optional[str]:
         return None
 
 
-# TODO(AIJUH): Add feature for argInfo processing and other scan type classes.
+# TODO(AIJUH): Add other scan type classes.
 class _ScanEntry(QWidget):
     """Entry class for a scannable object.
     
     Attributes:
         name: The name of the scannable object.
-        argInfo: The infomation of the arguments.
+        state: The dictionary that describes argument information of each scan type.
```

Comment:
I think you should write more description about the structure of `state`.
However, if you improve it in next PR, it is okay to remain it in this PR.

Reply:
I changed like below. Detailed info about each scannble type is described in each scan type Entry.
```
        state: Each key and its value are:
          "selected": The name of the selected scannable type.
          "NoScan": The dictionary that contains argument info of NoScan scannable type.
          "RangeScan": The dictionary that contains argument info of RangeScan scannable type.
          "CenterScan": The dictionary that contains argument info of CenterScan scannable type.
          "ExplicitScan": The dictionary that contains argument info of ExplicitScan scannable type.
```